### PR TITLE
Bugfix sprint09 tijn fix issue 637

### PR DESCRIPTION
**Pull request by tschmits**

### CODE REVIEW

File: app/scripts/components/scenariobuilder.js [hunk 1]
```diff
@@ -327,6 +329,9 @@ var exports = (function() {
       },
 
       submitScenario: function() {
+        if (this.totalRuns > this.maxRuns || !this.validForm) {
```
Comment (SiggyF) [inline app/scripts/components/scenariobuilder.js]: move totalRuns > maxRuns in validForm check?

File: app/templates/scenario-builder.html [hunk 2]
```diff
@@ -224,7 +224,7 @@ <h3 class="panel-title">{{section.name}}</h3>
                      id="scenario-submit"
                      @click="submitScenario"
                      value=""
-                     :class="{ 'disabled': !(totalRuns <= 20 && validForm) }" >
+                     :class="{ 'disabled': totalRuns > maxRuns || !validForm }" >
```
Comment (SiggyF) [inline app/templates/scenario-builder.html]: merge checks into 1 validForm computed property?

File: app/scripts/components/scenariobuilder.js (hunk 1 above)
Comment (tschmits) [inline app/scripts/components/scenariobuilder.js]: Good suggestion! Will do.

Comment (tschmits) [inline app/scripts/components/scenariobuilder.js]: Done, I like the solution. Thanks!

**Pull request merged**